### DISCUSSION TITLE
refactor(wdio): remove sync mode

### DISCUSSION THREAD

**Discussion by mohanraj-r** [salesforce/sa11y#58] [Tech Debt]
As of [Release v7.4.6 · webdriverio/webdriverio](https://github.com/webdriverio/webdriverio/releases/tag/v7.4.6) `sync` mode in WebdriverIO is being deprecated as the underlying node fibers package won't be working with newer versions of node. 

Remove the sync APIs and simplify code.

**Comment by mohanraj-r** [comment]
More sync related issues/resources: 
* https://github.com/webdriverio/webdriverio/issues/7381
* [Sync API Deprecation | WebdriverIO](https://webdriver.io/blog/2021/07/28/sync-api-deprecation/)

**Comment by jasonschroeder-sfdc** [comment]
Sync mode is gone.. https://github.com/salesforce/sa11y/pull/150
